src/components/carousel/index.tsx

```diff
@@ -212,6 +220,17 @@ const Carousel: FC<Props> = (props) => {
           paginationCarouselRef={paginationCarouselRef}
         />
       ) : null}
+
+      {mainCarousel && !fullScreen && withCustomPagination ? (
+        <Box paddingY="sm">
```

Comment:
this increases the size of the viewport since it's not part of the carousel. Could we check how it has been done for the ThumbnailPagination and align? Maybe it's also worth checking if it should be solved in the same way as the ThumbnailPagination since this manual onChange approach is not really following the docs: https://www.embla-carousel.com/examples/static/#arrows--dots

Reply:
it is also a bit confusing to me now that there are two chevrons now doing the same (next and previous slide)

Reply:
regarding two chevrons, this is my concern as well, I'm in touch with designers regarding that and will update when I get the answer

Reply:
Discussed with the designer, no need for chevrons there.